docs/content/primary-key-table/compaction.md

```diff
@@ -57,8 +57,8 @@ expect a mode to have maximum writing throughput, the compaction can be done slo
 You can use the following strategies for your table:
 
 ```shell
-num-sorted-run.stop-trigger = 2147483647
-sort-spill-threshold = 10
+num-sorted-run.stop-trigger = 10
```

Comment:
I think:
If apply this config, the commit will be block by compaction when the number of sorted-run is 10.
Then writing throughput will decline.
May I ask why you made this modification？